controllers/clusteroperator_controller.go

```diff
@@ -0,0 +1,69 @@
+/*
+Copyright 2021.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package controllers
+
+import (
+	"context"
+
+	"github.com/go-logr/logr"
+	configv1 "github.com/openshift/api/config/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	ctrl "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	// Lock in some nessesary libraries for operator functionality
+	_ "github.com/openshift/api/operator/v1"
+	_ "github.com/openshift/library-go/pkg/operator/events"
+	_ "github.com/openshift/library-go/pkg/operator/resource/resourceapply"
+	_ "github.com/openshift/library-go/pkg/operator/resource/resourcehash"
+	_ "github.com/openshift/library-go/pkg/operator/resource/resourcemerge"
```

Comment:
These aren't being used? Should we drop them until they are being used? What's the benefit of doing this? What if they never get used?

Reply:
I was trying to lock up `library-go` for now. But putting a clusterOpreator template logic already did it, so it is no longer needed.